stores/flatten.go

```diff
@@ -60,6 +61,28 @@ func Flatten(in map[string]interface{}) map[string]interface{} {
 	return newMap
 }
 
+func Flatten(md Metadata) (map[string]interface{}, error) {
```

Comment:
Instead, this function should be called `FlattenMetadata`.

Reply:
done